### DISCUSSION TITLE
Cannot read properties of undefined (reading 'getBoundingClientRect')

### DISCUSSION THREAD

**Discussion by mohaahadji** [fabricjs/fabric.js#9499] [General]
When I try to scale the IText I get this error "Cannot read properties of undefined (reading 'getBoundingClientRect')" and it's happening only with text not images and shapes. 
<img width="726" alt="Screenshot 2566-11-19 at 14 48 43" src="https://github.com/fabricjs/fabric.js/assets/59908103/c9997f1a-e079-4bb3-a967-acb493b4ae15">
Anyone can help guide me to the solution of this issue will be much appreciated?

**Comment by ShaMan123** [comment]
File a bug report for an exception
Seems you are working on v5?
Anyways I recall we fixed this on v6. Upgrade and see if it resolves

**Reply by mohaahadji** [reply]
Yes I am working with v5! the error still persisting.. I will open an issue and try v6 as well! Thanks!
